tools/debug-ui/src/LatestBlocksView.tsx

```diff
@@ -207,9 +210,8 @@ const BlocksTable = ({ rows, knownProducers, expandAll, hideMissingHeights }: Bl
         tableRows.push(
             <tr
                 key={block.block_hash}
-                className={`block-row ${
-                    row.block.is_on_canonical_chain ? '' : 'not-on-canonical-chain'
-                }`}>
+                className={`block-row ${row.block.is_on_canonical_chain ? '' : 'not-on-canonical-chain'
+                    }`}>
```

Comment:
Formatting ended up pretty weird here, but from what I can tell there is no other change here from what it was before? I'd say either undo or put everything on a single line.